src/main/java/com/autotune/analyzer/experiment/ExperimentInterfaceImpl.java

```diff
@@ -122,4 +125,36 @@ public boolean addResultsToLocalStorage(Map<String, KruizeObject> mainKruizeExpe
         return true;
     }
 
+    @Override
+    public boolean addRecommendationsToLocalStorage(Map<String, KruizeObject> mainKruizeExperimentMap,
```

Comment:
Can you use this function while generating recommendation for first time . Currently it is getting stored without this function , can you use this function there as well.  addRecommendationsToLocalStorage should be called at two place